geolocator_android/lib/src/types/foreground_settings.dart

```diff
@@ -0,0 +1,74 @@
+/// Uniquely identifies an Android resource
```

Comment:
Nit: sentence should end with a period.
```suggestion
/// Uniquely identifies an Android resource.
```